whois-nrtm4/src/test/resources/nrtm_sample_sm.sql

```diff
@@ -2,7 +2,7 @@ INSERT INTO `history` VALUES (5158,1,1355413825,2,'aut-num:         AS6\nas-name
 INSERT INTO `history` VALUES (5158,2,1335191471,2,'aut-num:         AS6\nas-name:         ASNAME\ndescr:           Description\norg:             ORG-TEST-RIPE\nadmin-c:         TEST-RIPE\ntech-c:          TEST-RIPE\nmnt-by:          RIPE-NCC-TEST-MNT\nmnt-by:          TEST-MNTNR\nsource:          RIPE\n','AS6');
 
 INSERT INTO `last` VALUES (5158,0,1355413825,2,'','AS6');
-INSERT INTO `last` VALUES (11044887,1,1335191516,6,'inetnum:         195.77.187.144 - 195.77.187.151\nnetname:         Netname\ndescr:           Description\ncountry:         es\nadmin-c:         TEST-RIPE\ntech-c:          TEST-RIPE\nstatus:          ASSIGNED PA\nmnt-by:          MAINT-AS3352\nsource:          RIPE\n','195.77.187.144 - 195.77.187.151');
+INSERT INTO `last` VALUES (11044887,1,1335191516,6,'inetnum:         195.77.187.144 - 195.77.187.151\nnetname:         Netname\ndescr:           Descrip-\\ntion\ncountry:         es\nadmin-c:         TEST-RIPE\ntech-c:          TEST-RIPE\nstatus:          ASSIGNED PA\nmnt-by:          MAINT-AS3352\nsource:          RIPE\n','195.77.187.144 - 195.77.187.151');
```

Comment:
Can we use DatabaseHelper to add delete objects instead of insert statements? Or is it necessary here? It's just you do not need to manually add entries in history table and other tables. It makes it easier